test/functional/feature_assumeutxo.py

```diff
@@ -133,13 +133,20 @@ def expected_error(msg):
         cases = [
             # (content, offset, wrong_hash, custom_message)
             [b"\xff" * 32, 0, "7d52155c9a9fdc4525b637ef6170568e5dad6fabd0b1fdbb9432010b8453095b", None],  # wrong outpoint hash
-            [(2).to_bytes(1, "little"), 32, None, "Bad snapshot data after deserializing 1 coins."],  # wrong txid coins count
+            [(2).to_bytes(1, "little"), 32, None, "Bad snapshot data after deserializing 1 coins"],  # wrong txid coins count
```

Comment:
nit `1 coin`?

Reply:
This is an expected error from the underlying RPC call. Although using `coin` here in the assertion won't fail the functional test but I don't think this makes any meaningful difference.

Reply:
Ah, I see thanks.  I guess the underlying call would be more clear to return `coin(s)` although that's besides the point.